apps/crn-server/src/controllers/reminder.controller.ts

```diff
@@ -42,7 +42,7 @@ export default class ReminderController {
             href: sharedResearch({}).researchOutput({
               researchOutputId: reminder.data.researchOutputId,
             }).$,
-            description: `${reminder.data.title} ${reminder.data.documentType} from your ASAP Team is now published on the Hub. If there are errors, please let your PM know.`,
+            description: `**${reminder.data.statusChangedBy}** on ${reminder.data.associationType} **${reminder.data.associationName}** published a ${reminder.data.associationType} output: ${reminder.data.title}.`,
```

Comment:
the output type is missing here
![Screenshot 2023-08-02 at 09 41 57](https://github.com/yldio/asap-hub/assets/16595804/3ff37e02-2c9b-4dc9-99a2-cb4c76eaf496)